File: FunnyAlgorithms-master/sort/bubbleSort/BubbleSort.java
import java.util.Random;

class BubbleSort{

	public static void bubbleSort(int arr[]){

		int temp;

		for(int i=0; i<arr.length; i++){
			for(int j=0; j<arr.length-i-1; j++){
				if( arr[j] > arr[j+1] ){
					temp = arr[j];
					arr[j] = arr[j+1];
					arr[j+1] = temp;
				}
			}
		}
	}

	static void printArray(int arr[]){
		for(int x : arr)
			System.out.print(x + ", ");
	}

	public static void main(String args[]){

		int arr[] = new int[10];
		Random r = new Random();

		for(int i=0; i<10; i++){
			arr[i] = r.nextInt(100);
		}

		System.out.print("Orignal array : ");
		printArray(arr);

		bubbleSort(arr);

		System.out.print("\n Sorted array : ");
		printArray(arr);

		System.out.println();
	}
}

File: algorithms/sort/bubbleSort/Bubble_Sort.py
from random import randint

def Bubble_Sort(A):
    Flag = 1
    while Flag == 1 :
        Flag = 0
        for j in range(len(A)-1):
            if A[j] > A[j+1] :
                A[j], A[j+1] = A[j+1], A[j]
                Flag = 1

A = [randint(-100,100) for i in range(10)]
print("Orignal array = ", *A)
Bubble_Sort(A)
print(" Sorted array = ", *A)
